package.json

```diff
@@ -55,7 +55,8 @@
   },
   "license": "BSD-3-Clause",
   "dependencies": {
-    "node": "10.2.0"
+    "node": "12.4.0",
```

Comment:
VSCode ships with node 10.11.0 in version 1.39 before that they were shipping with node 10.2.0. In the latest vscode versions (1.40 and above) they moved to node 12.4.0 starting on version 1.40